### PR TITLE
Add win/add score count

### PR DESCRIPTION
**Pull request by cairnsr**
Readded the score counter so all should be working properly now

### CODE REVIEW

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -16,83 +16,70 @@
...
-          }
+        for (int i = 0; i < 4; i++) {   //loop through all 4 cols
+
+            if (columnHasCards(i)) {
+                Card checkCard = cols.get(i).getTopCard();    //take top card of that col
+                if (Objects.equals(removableCard.suit, checkCard.suit) && removableCard.value < checkCard.value) {   //compare card suit and vals
+                    cols.get(columnNumber).removeTopCard();  //if suit match and val less then its removeable
+                    this.score++; //adds to score every time card is removed
+                    if (this.score == 48) //if score = max score then print win message
+                        JOptionPane.showMessageDialog(null, "Congradulations on this Huge Accomplishment", "You Win!! ", JOptionPane.INFORMATION_MESSAGE);
```
Comment (elitucker) [inline src/main/java/models/Game.java]: Oops! Change that 'd' to a 't' in Congratulations. Other than that, it seems as if the code is up to scratch. As long as it looks good to everyone else, we can fix that one issue than merge the request.

**Comment by cairnsr** [comment]
Alright I fixed the typo so this should be good to go

**Comment by terri-hewitt** [comment]
This code looks good to me. I like that you changed the name of the class "score" to "scoreContainer" in the CSS. That definitely helps to clarify specifically what the code in that class is supposed to be styling and makes the code more readable.

**Comment by elitucker** [comment]
This is all good. I am going to merge with master.

**Pull request merged**